lib/concurrent/synchronization/lockable_object.rb

```diff
@@ -13,7 +13,7 @@ module Synchronization
                                    when Concurrent.on_rbx?
                                      RbxLockableObject
                                    when Concurrent.on_truffle?
-                                     TruffleLockableObject
+                                     MriMutexLockableObject
```

Comment:
I don't get any spec failures (or I do but they're intermittent) so I assumed that whatever `object.rb` is doing must be ok at the moment.